EST.MIT.Payment.Function/Validation/ValidateRequest.cs

```diff
@@ -168,14 +157,17 @@ public static class ValidateRequest
             }
         },
         ""required"": [
-            ""invoices""
+            ""paymentRequestsBatches""
         ]
         }";
 
     public static bool IsValid(string importRequest)
     {
         var schema = JSchema.Parse(schemaJson);
         var parseImportRequest = JObject.Parse(importRequest);
+
+        parseImportRequest.IsValid(schema);
```

Comment:
This looks a bit odd. I think the parseImportRequest that doesn't get returned can be removed?

Reply:
Resolved